scalafmt-tests/src/test/resources/unit/Dialect.stat

```diff
@@ -7,7 +7,7 @@ foo(
 >>>
 foo(
     1,
-    2,
+    2
```

Comment:
This PR is currently breaking. I have an idea on how to make this non-breaking by default.